### PR TITLE
catch status 401

### PR DESCRIPTION
**Pull request by xlogex**

### CODE REVIEW

File: src/Client/Shared/ApiHelper.cs [hunk 1]
```diff
@@ -38,6 +39,13 @@ public static class ApiHelper
         {
             snackbar.Add(ex.Result.Exception, Severity.Error);
         }
+        catch (ApiException ex)
```
Comment (fretje) [inline src/Client/Shared/ApiHelper.cs:42]: Where/when exactly do you get into this catch block without it being catched by the ApiException<ErrorResult> block before? The api should always send an errorresult... Or maybe it's in the case of not even getting to the api? Still... when does this happen?

Comment (xlogex) [reply src/Client/Shared/ApiHelper.cs:42]: must i close this issue/pr?

Comment (fretje) [reply src/Client/Shared/ApiHelper.cs:42]: Not necessarily... We should find the actual reason why you're having this problem...

How exactly do you repeat this? Tell the exact steps you take.

Comment (xlogex) [reply src/Client/Shared/ApiHelper.cs:42]: this front-end (current version) cannot deserialize `ApiException<ErrorResult>` lost so catch

Comment (fretje) [reply src/Client/Shared/ApiHelper.cs:42]: Yes but *why* can it not deserialize it? And *when* exactly does this happen? **Which exact steps did you take to get here?**

Comment (xlogex) [reply src/Client/Shared/ApiHelper.cs:42]: for example 
image

Comment (xlogex) [reply src/Client/Shared/ApiHelper.cs:42]: image

Comment (xlogex) [reply src/Client/Shared/ApiHelper.cs:42]: when you send a wrong field then api responde with status different to 200 with exception message

**Comment by xlogex** [comment]
https://github.com/fullstackhero/dotnet-webapi-boilerplate/pull/375

**Pull request closed without merging**